qu.json

```diff
@@ -0,0 +1,192 @@
+[
```

Comment:
I changed the name to data.questions.json, delete this file so there's no duplicate